athena-dynamodb/athena-dynamodb.yaml

```diff
@@ -24,6 +24,20 @@ Parameters:
     Description: 'The prefix within SpillBucket where this function can spill data.'
     Type: String
     Default: athena-spill
+  SpillPutRequestHeaders:
+    Description: '(Optional) A JSON encoded map of request headers and values for the Amazon S3 putObject request that is used for spilling. (for example, {"x-amz-server-side-encryption" : "aws:kms"}).'
+    Type: String
+  DisableGlue:
+    Description: "(Optional) If present and set to 'true', the connector does not attempt to retrieve supplemental metadata from AWS Glue."
+    Default: 'false'
+    Type: String
+  GlueCatalog:
+    Description: '(Optional) Use this option to specify the account id of a different account for a cross-account AWS Glue catalog. By default, the connector attempts to get metadata from its own AWS Glue account.'
+    Type: String
+  DisableProjectionAndCasing:
+    Description: "(Optional) Disables projection and casing. Use if you want to query DynamoDB tables that have casing in their column names and you do not want to specify a columnMapping property on your AWS Glue table. Can be set to 'auto' or 'always', set to 'auto' by default. See documentation for more information."
+    Default: 'auto'
+    Type: String
```

Comment:
Add allowedValues to provided validation on input
    AllowedValues:
      - auto
      - always